modules/490-virtualization/templates/image/ubuntu.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
mb templates/images?

Reply:
fixed in 4a59b7a7185d4a3c864bcaaf329cd154ab2685de